gcsweb/Makefile

```diff
@@ -0,0 +1,150 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The binary to build (just the basename).
+BIN := gcsweb
+
+# This repo's root import path (under GOPATH).
+PKG := k8s.io/test-infra/gcsweb
+
+# Where to push the docker image.
+REGISTRY ?= gcr.io/google_containers
```

Comment:
We can subdir undir gcr.io/google_containers, too.  Though this is a pretty generic container - I would not at all be surprised if people in the wild wanted to use it.